config/initializers/elasticsearch.rb

```diff
@@ -1,9 +1,37 @@
-config = {
-  host: ENV.fetch("ES_HOST", "http://127.0.0.1:9200"),
-  transport_options: {
-    request: { timeout: 5 }
+require 'base64'
+
+vcap_services = ENV.fetch("VCAP_SERVICES", nil)
```

Comment:
`fetch` is useful if we want it to blow up if the key does not exist, or if we want to offer a fallback value. In this case, we could just write `ENV['VCAP_SERVICES']`, which will return `nil` if the `VCAP_SERVICES` key is not present in the `ENV` hash.